src/templates/policies.bicep

```diff
@@ -185,6 +185,8 @@ var loadPolicyDefinitions = {
     loadTextContent('../resources/Microsoft.Authorization/policyDefinitions/Deploy-MDFC-SQL-AMA.json')
     loadTextContent('../resources/Microsoft.Authorization/policyDefinitions/Deploy-MDFC-SQL-DefenderSQL-DCR.json')
     loadTextContent('../resources/Microsoft.Authorization/policyDefinitions/Deploy-MDFC-SQL-DefenderSQL.json')
+    loadTextContent('../resources/Microsoft.Authorization/policyDefinitions/Deploy-Private-DNS-GroupID.json')
```

Comment:
```suggestion
```

Reply:
Removed the text from policies.bicep file